lib/BackgroundScanner.php

```diff
@@ -189,6 +189,10 @@ protected function scanOneFile($owner, $fileId) {
 		$this->initFilesystemForUser($owner);
 		$view = Filesystem::getView();
 		$path = $view->getPath($fileId);
+		\OC::$server->getLogger()->info(
+			"About to scan file of a user {$owner} with id {$fileId} and path {$path}",
```

Comment:
@VicDeo if you do a change I propose a slight text adoption:
`About to scan file of a user` -->  `About to scan file of user`
(No a)
And if you have the filename it helpful for the admin.